### ISSUE TITLE
gdb-server: L0xx has no FP_CTRL register for breakpoints

### ISSUE THREAD

**Issue by ghost** [stlink-org/stlink#273]
There is no FP_CTRL register on the L0xx. In gdb-server.c, you have:

```
static void init_code_breakpoints(stlink_t *sl) {
    memset(sl->q_buf, 0, 4);
    stlink_write_debug32(sl, CM3_REG_FP_CTRL, 0x03 /*KEY | ENABLE4*/);
    if(...){
        ELOG("[FP_CTRL] = 0x%08x expecting 0x%08x\n", val,
    ...
    }
    ...
}
```

This causes the following error in st-util output:

```
$ ./st-util
2014-10-14T13:33:23 INFO src/stlink-common.c: Loading device parameters....
2014-10-14T13:33:23 INFO src/stlink-common.c: Device connected is: L0x3 device
...
2014-10-14T13:33:23 INFO gdbserver/gdb-server.c: Listening at *:4242...
2014-10-14T13:33:39 ERROR gdbserver/gdb-server.c: [FP_CTRL] = 0x00000041 expecting 0x00000261
```

The FPB and its registers have been replaced by the BPU on the M0 chips. Please see [the ARM Info Center](http://infocenter.arm.com/help/topic/com.arm.doc.ddi0432c/DDI0432C_cortex_m0_r0p0_trm.pdf) and its section on BPU functionality in the Debug section of the "Cortex M0 Technical Reference Manual".

**Comment by xor-gate** [comment]
Should work with latest 1.4.0 release.

**Comment by Nightwalker-87** [comment]
Closed by #341.
